docs/operations/faq/zone_aware_routing.rst

```diff
@@ -0,0 +1,47 @@
+.. _common_configuration_zone_aware_routing:
+
+Zone aware routing
+==================
+
+There are several steps required for enabling :ref:`zone aware routing <arch_overview_load_balancing_zone_aware_routing>`
+between source service A and destination service B.
+
+Envoy configuration on source service
+-------------------------------------
+* :ref:`sds <config_cluster_manager_type>` type must be used for cluster B definition.
+  See more on service discovery :ref:`here <arch_overview_service_discovery_sds>`.
+* sds type must be used for cluster A definition.
+* :ref:`local_cluster_name <config_cluster_manager_local_cluster_name>` must be set to cluster A.
+  Only essential part is listed in the configuration below.
+
```

Comment:
I think this could be rephrased to be a little more clear. 

Both cluster A and cluster B must have type sds. Seem more on service...